.changes/README.md

```diff
@@ -0,0 +1,46 @@
+# Changes
+
+This stores changes to automatically generate the changelog, to avoid merge conflicts. Files should be in a JSON format, with the following format:
+
+```json
+{
+    "description": "single-line description to add to the CHANGELOG.",
+    "issues": [894],
+    "type": "added",
+    "breaking": false
+}
+```
+
+Valid types are:
+- added (Added)
+- changed (Changed)
+- fixed (Fixed)
+- removed (Removed)
+- internal (Internal)
+
+`breaking` is optional and defaults to false. if `breaking` is present for any active changes, we will not allow a minor (or pre-1.0) patch version to be generated. `issues` is also optional, and is currently unused, and is an array of issues fixed by the PR, and defaults to an empty array.
+
+The file numbers should be `${pr}.json`
```

Comment:
It's not too bad I think, if it has the issue number since we can just choose to report those. We would need to write them at the end, however, since they won't sort well.

Reply:
Done: PR numbers are now optional, and they are added to the bottom, sorted by description.